keylime/revocation_notifier.py

```diff
@@ -72,7 +72,7 @@ def stop_broker():
         logger.info("Stopping revocation notifier...")
         broker_proc.terminate()
         broker_proc.join(5)
-        if broker_proc.is_alive():
+        if broker_proc.is_alive() and hasattr(broker_proc, "kill"):
```

Comment:
I would use

```python
if sys.version_info >= (3, 7):
    logger.debug("Killing revocation notifier because it did not terminate after 5 seconds...")
    broker_proc.kill()
```